skopt/utils.py

```diff
@@ -0,0 +1,20 @@
+import numpy as np
+from scipy.optimize import OptimizeResult
+
+def pack_optimize_result(Xi, yi, space, rng, specs, models=None):
```

Comment:
Shall we aim for a shorter function name? e.g. simply `pack`? (which should be clear from the context)